integration_tests/__tests__/failures-test.js

```diff
@@ -17,6 +17,25 @@ const normalizeDots = text => text.replace(/\.{1,}$/gm, '.');
 
 skipOnWindows.suite();
 
+const cleanupStackTrace = stderr => {
```

Comment:
Can we make sure this function verifies that there is a stack trace for both and fails if one of them or both are missing? :)